### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maks2809**
[DEMO LINK](https://maks2809.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://maks2809.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,65 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo"
+          class="logo__image">
```
Comment (Andriy-Shymkiv) [inline src/index.html:19]:
```suggestion
        <img 
          src="./images/logo.png" 
          alt="Moyo logo"
          class="logo__image"
        >
```

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,65 @@
...
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >
+              LAPTOPS &amp; COMPUTERS
```
Comment (Andriy-Shymkiv) [inline src/index.html:42]:
```suggestion
              Laptops &amp; Computers
```

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,65 @@
...
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo"
+          class="logo__image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
```
Comment (Andriy-Shymkiv) [inline src/index.html:25]: add more meaningful href's to your links

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
```
Comment (Andriy-Shymkiv) [inline src/style.css:10]: it is white by default

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:23]: you don't need so many styles for logo, please remove some of them (use devtools)

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,71 @@
...
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
```
Comment (Andriy-Shymkiv) [inline src/style.css:43]: don't think that it is a good idea to set fixed height for link. maybe `padding` will be better? what do you think?

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,71 @@
...
+  font-size: 12px;
+  text-decoration: none;
+  white-space: nowrap;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (Andriy-Shymkiv) [inline src/style.css:54]: move this to a separate variable

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 6 above)
Comment (maks2809) [reply src/style.css:43]: I did not fully understand what you mean, can you explain in more detail, please?

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,66 @@
...
   margin: 0;
+  --link-hover-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
```
Comment (denys-danyliuk) [inline src/style.css:11]: Знімок екрана 2023-08-10 о 12 08 54

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,66 @@
...
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  height: 60px;
```
Comment (denys-danyliuk) [inline src/style.css:39]: use paddings instead

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,66 @@
...
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:38]: 3'rd point in checklist

File: src/index.html [hunk 11]
```diff
@@ -1,14 +1,68 @@
...
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+          >
```
Comment (denys-danyliuk) [inline src/index.html:22]:
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo logo"
          class="logo__image"
        >
```
closing bracket should be on the same level as opening one

File: src/index.html [hunk 12]
```diff
@@ -1,14 +1,68 @@
...
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo" class="logo">
```
Comment (denys-danyliuk) [inline src/index.html:17]: it should be link to the main page
```suggestion
      <a href="#home" class="logo">
```

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**